app/serializers/api/enterprise_shopfronts_serializer.rb

```diff
@@ -0,0 +1,53 @@
+# Represents the minimum details of an Enterprise when all shopfronts are being listed
```

Comment:
ah, now I get it :-) this is the serializer for the shoplist
EnterpriseShoplistSerializer?

Reply:
Yes, so we have:
1. A _thin_ representation of a shopfront for listing all shopfronts
2. A fatter representation of the details of a single shopfront
3. An even thinner representation of related producers or hubs, which is nested in `2`.

Reply:
@luisramos0 I went with: 
1. EnterpriseShopfrontsSerializer 
2. EntrpriseShopfrontSerializer
3. EnterpriseThinSerializer

Does that sound ok? I've moved the API endpoint as well and scrapped the `api/shop_controller`.

Reply:
yeah, awesome!
I dont get much context from "shopfronts" so I prefer EnterpriseShoplistSerializer

but with that loadtime improvement (40s - 1 sec) we will approve the PR even with a BananasSerializer ;-)

Reply:
I've changed it to: `EnterpriseShopfrontListSerializer`

Reply:
awesome!